packages/patternfly-4/react-core/src/components/LoginPage/LoginForm.test.js

```diff
@@ -74,3 +74,21 @@ test('LoginForm with rememberMeLabel and rememberMeAriaLabel does not generates
   expect(view).toMatchSnapshot();
   expect(myMock).not.toBeCalled();
 });
+
+test('should call onChangeUsername when is set and username is changed', () => {
+  const mockFn = jest.fn();
+  const view = shallow(
+    <LoginForm onChangeUsername={mockFn} rememberMeLabel="Login Form" rememberMeAriaLabel="Login Form" />
+  );
+  view.find('#pf-login-username-id').simulate('change');
+  expect(mockFn).toHaveBeenCalled();
+});
+
+test('should call onChangeText callback when is set and password is changed', () => {
+  const mockFn = jest.fn();
+  const view = shallow(
+    <LoginForm onChangePassword={mockFn} rememberMeLabel="Login Form" rememberMeAriaLabel="Login Form" />
+  );
+  view.find('#pf-login-password-id').simulate('change');
+  expect(mockFn).toHaveBeenCalled();
+});
```

Comment:
should probably add a test case for when the user defines both onChangeUsername/onChangePassword and OnChangeText.

Reply:
You're right. Will do